javascript/javascript1/week1/nadiia/Goodboy-Oldboy.js

```diff
@@ -0,0 +1,30 @@
+function dogAgeIfy(){
+
+    let dogYearOfBirth = parseInt(document.getElementById("dogYearOfBirth").value, 10);
+    let dogYearFuture = parseInt(document.getElementById("dogsFutureYear").value, 10);
+    let dogYear = (dogYearFuture - dogYearOfBirth) * 7;
+
+    document.getElementById("dogFutureAge").innerText = "Your dog will be " + dogYear + " dog years old in " + dogYearFuture + ".";
+
+}
+
+function dogAgeIfyInHuman(){
+
+    let dogYearOfBirth = parseInt(document.getElementById("dogYearOfBirth").value, 10);
+    let dogYearFuture = parseInt(document.getElementById("dogsFutureYear").value, 10);
+    let dogYear = dogYearFuture - dogYearOfBirth;
+
+    document.getElementById("dogFutureAge").innerText = "Your dog will be " + dogYear + " human years old in " + dogYearFuture + ".";
+}
+
+/* With if/else statement:
+let shouldShowResultInDogYears = true; //or false
+
+if(shouldShowResultInDogYears === true){
```

Comment:
Generally, you don't need to add `=== true` when using a boolean value in conditional logic.
```suggestion
if(shouldShowResultInDogYears) {
```
would be enough